### DISCUSSION TITLE
PPPOE Chap authentication failed

### DISCUSSION THREAD

**Discussion by yhy660** [accel-ppp/accel-ppp#211] [General]
PPPOE Chap authentication failed
How to configure the configuration file?
My challenge response is MD5 (ID+secret+challenge), right?
![pppoe_conf](https://github.com/user-attachments/assets/20a24e81-b227-4810-b166-b768b9a468d3)
![chap-secrets](https://github.com/user-attachments/assets/9d7a96e4-3490-45d8-ba6e-a5e4d5b20110)
![ppp-chap-secrets](https://github.com/user-attachments/assets/f4831584-2872-456c-a641-26c2dd3d87cc)

[pppoe_chap.zip](https://github.com/user-attachments/files/17405670/pppoe_chap.zip)

**Comment by Tokarak** [comment]
May be related to #189
